app/views/transcripts/charts/_bubble_courses.html.haml

```diff
@@ -10,6 +18,7 @@
 
       // Set chart options
       var options = {
+        tooltip: { isHtml: true }, //It will make the chart tooltip be html rendered instead of svg rendered
```

Comment:
Thank you @YgorCr, we thank you for the contribution. Great thought about the issue, and NP about the build.